tasks/Divergent Association Task.md

```diff
@@ -0,0 +1,54 @@
+# Summary
+Measuring creativity
+Participants are asked to come up with 10 nouns that are as different as possible.
+
+# References
+Olson, Jay A., et al. “Naming Unrelated Words Predicts Creativity.” PNAS, National Academy of Sciences, 22 June 2021, www.pnas.org/content/118/25/e2022340118. 
+
+Link: https://www.pnas.org/content/118/25/e2022340118
+
+# Stimuli
+## The visual components
+N/A
```

Comment:
![image](https://user-images.githubusercontent.com/78745728/131905170-0cfc2cac-df43-49ae-9adb-72b39874c26a.png)
Can you add this to the image folder? it's in this document: https://www.pnas.org/content/pnas/suppl/2021/06/17/2022340118.DCSupplemental/pnas.2022340118.sapp.pdf